src/components/markdown/RuleFormattedField.tsx

```diff
@@ -0,0 +1,44 @@
+interface FormattedFieldProps {
+  fKey: string;

```

Comment:
`fKey` is not a descriptive name.

Also if there are only a handful of valid values, it should be an enum.